test/main.hs

```diff
@@ -28,6 +38,28 @@ main = bracket (connect testConn) close $ \conn -> hspec $ do
     describe "Database" $ do
       it "seems to be connected" $ do
         query conn "select 1 + 1"
-        result <- useResult conn
+        result <- storeResult conn
```

Comment:
when using `useResult` the connection would be left in an unusable state and prepare would fail. By switching to `storeResult` this issue went away.